src/index.js

```diff
@@ -0,0 +1,131 @@
+// ------------- Wave 2 ----------------------
+const state = {
+  temp: 60,
+  city: 'Seattle',
+  sky: '☁️',
+};
```

Comment:
`sky` is a bit inconsistent with the rest of your code here. In the rest of your code, you use words like "sunny" instead of the emoji directly.